docs/core/whats-new/dotnet-core-3-0.md

```diff
@@ -543,9 +504,19 @@ The `MetadataLoadContext` has a resolver class passed to its constructor. The re
 
 The [MetadataLoadContext tests](https://github.com/dotnet/corefx/tree/master/src/System.Reflection.MetadataLoadContext/tests/src/Tests) demonstrate many use cases. The [Assembly tests](https://github.com/dotnet/corefx/blob/master/src/System.Reflection.MetadataLoadContext/tests/src/Tests/Assembly/AssemblyTests.cs) are a good place to start.
 
+
+## Type: AssemblyDependencyResolver
+
+Loading dependent assemblies from a given path was not supported in .NET Core. Now with the <xref:System.Runtime.Loader.AssemblyDependencyResolver> type, you can help `AssemblyLoadContext` resolve paths to dependent assemblies. For code examples, see the [AppWithPlugin Demo](https://github.com/dotnet/samples/tree/master/core/extensions/AppWithPlugin).
+
+## Type: NativeLibrary
+
+<xref:System.Runtime.InteropServices.NativeLibrary> provides an encapsulation for loading a native library (using the same load logic as .NET Core P/Invoke) and providing the relevant helper functions such as `getSymbol`. For a code example, see the [DLLMap Demo](https://github.com/dotnet/samples/tree/master/core/extensions/AppWithPlugin).
+https://github.com/dotnet/samples/tree/master/core/extensions/DllMapDemo
+
 ## TLS 1.3 & OpenSSL 1.1.1 on Linux
 
-.NET Core will now take advantage of [TLS 1.3 support in OpenSSL 1.1.1](https://www.openssl.org/blog/blog/2018/09/11/release111/), when it is available in a given environment. There are multiple benefits of TLS 1.3, per the [OpenSSL team](https://www.openssl.org/blog/blog/2018/09/11/release111/):
+.NET Core now takes advantage of [TLS 1.3 support in OpenSSL 1.1.1](https://www.openssl.org/blog/blog/2018/09/11/release111/), when it's available in a given environment. There are multiple benefits of TLS 1.3, per the [OpenSSL team](https://www.openssl.org/blog/blog/2018/09/11/release111/):
 
 * Improved connection times due to a reduction in the number of round trips required between the client and server.
 
```

Comment:
line 525 is still talking about Preview 1. Time for an update?

is capable of utilizing -> can use